docs/index.md

```diff
@@ -308,10 +308,7 @@ print(Settings().model_dump())
     'sub_model': {'v1': 'json-1', 'v2': b'nested-2', 'v3': 3, 'deep': {'v4': 'v4'}},
 }
 """
-```
-
-1. Sub model has to inherit from `pydantic.BaseModel`, Otherwise `pydantic-settings` will initialize sub model,
```

Comment:
We should keep this one and change the next one to `2. ` because we have a number `# (2)!` in the above code.

Reply:
see ec82f7e47a3fb67a5c112582731c541344ddc240